src/main/java/org/jala/efeeder/tyche/TycheCommand.java

```diff
@@ -48,11 +48,13 @@
 			"SELECT name, last_name FROM user WHERE id = ?";
 	private static final String SET_BUYER_DICE =
 			"INSERT INTO buyer(id_food_meeting, id_user, dice) VALUES(?, ?, ?)";
+	
+	private static final Integer RANGE = (999999999 - 100000000) + 1;
```

Comment:
why we need this operation ?
I think it is more readable if you set the result of the operation